vhdbuilder/packer/post-install-dependencies.sh

```diff
@@ -31,6 +32,11 @@ if [[ $OS == $UBUNTU_OS_NAME ]]; then
   apt-get -y autoclean || exit 1
   apt-get -y autoremove --purge || exit 1
   apt-get -y clean || exit 1
+elif [[ $OS == $MARINER_OS_NAME ]]; then
+  current_kernel_version="$(uname -r | cut -d. -f-4)"
+  kernel_packages_to_remove=$(rpm -qa | grep "kernel" | grep -v $current_kernel_version)
+  dnf -y autoremove $kernel_packages_to_remove
+  dnf -y autoremove # remove all other unused packages
```

Comment:
Do we have a list of packages that `dnf -y autoremove` is uninstalling?

Reply:
I placed this here as a 'catch all', any that are removed should be printed to stdout by the dnf -y autoremove command.